AuroraPatch/Loader.cs

```diff
@@ -170,6 +173,28 @@ private void MapShown(object sender, EventArgs e)
                 }
             }
             Program.Logger.LogInfo("Done running Started");
+
+            // At this point Aurora is definitely opened, so "started" flag needs to be set to properly to indicate that from now on close button shall not terminate the application, but only hide form.
+            // Start monitoring Aurora, if the game is closed the application will also terminate.
+            started = true;
+            Thread thread = new Thread(MonitorAuroraProcess);
+            thread.Start();
+        }
+
+
+        // Check if Aurora is opened - if not, close the application.
+        internal void MonitorAuroraProcess()
+        {
+            while (true)
+            {
+                // Close AuroraPatch if tactical map is not visible or null.
+                if (TacticalMap is null || !TacticalMap.Visible)
```

Comment:
This should probably check if the AuroraPatch form has been closed. If the AuroraPatch form is still open then the application shouldn't exit. A user might want to close Aurora, change patches and/or their settings, and then start Aurora again.